pkg/cloud/aws/commands/run.go

```diff
@@ -142,7 +142,12 @@ func Run(ctx context.Context, opt flag.Options) error {
 		})
 	}
 
-	r := report.New(cloud.ProviderAWS, opt.Account, opt.Region, results.GetFailed(), opt.Services)
+	res := results.GetFailed()
+	if opt.MisconfOptions.IncludeNonFailures {
+		res = results
+	}
```

Comment:
Looks like the flag wasn't used in Cloud scanning, I've added it here. The results will still only be shown in the JSON output.